spyderlib/plugins/inspector.py

```diff
@@ -749,6 +749,8 @@ def show_intro_message(self):
             tutorial = _("tutorial")
             intro_message = intro_message % ("<b>Ctrl+I</b>", "<br><br>",

```

Comment:
Why not add a line before this one with something like

```
if sys.platform == 'darwin':
    shortcut = "<b>Cmd+I</b>"
else:
    shortcut = "<b>Ctrl+I</b>"
```

and then change this line like this

```
             intro_message = intro_message % (shortcut, "<br><br>", "<i>"+prefs+"</i>")
```

I think this approach is much cleaner than the proposed one.